src/steps/companies/__snapshots__/converters.test.ts.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`#createCompanyEntity should convert to entity 1`] = `
+Object {
+  "_class": Array [
+    "Organization",
+  ],
+  "_key": "hubspot_company:84106264 ",
+  "_rawData": Array [
+    Object {
+      "name": "default",
+      "rawData": Object {
+        "companyId": 84106264,
+        "isDeleted": false,
+        "portalId": 21406264,
+        "properties": Object {
+          "domain": Object {
+            "value": "awesomecompanydomain.tld",
+          },
+          "name": Object {
+            "value": "Awesome company",
+          },
+          "website": Object {
```

Comment:
Can we add the following properties from the `Organization` data model for consistency:

```
website
external
```

See: https://github.com/JupiterOne/data-model/blob/master/src/schemas/Organization.json